src/features/pastelDB/pastelDBThread.tsx

```diff
@@ -0,0 +1,227 @@
+import { remote } from 'electron'
```

Comment:
Let's try to add tests here. An example is here: https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/features/loading/__tests__/utils.test.ts

Reply:
- Added tests.